src/pages/Staking/index.tsx

```diff
@@ -663,8 +698,8 @@ const Staking = (): JSX.Element => {
               voteStakedAmount={renderVoteStakedAmountLabel()}
               rewardAmount={renderRewardAmountLabel()} />
             <ClaimRewardsButton
-              disabled={claimRewardsButtonEnabled === false}
-              pending={claimRewardsButtonEnabled === undefined} />
+              rewardsToClaim={renderRewardsToClaimLabel()}
```

Comment:
`claimableRewardAmount` and renderClaimableRewardAmountLabel` could be better for harmonized naming purpose I think.

Reply:
yeah. happy to have this changed

Reply:
I updated them.